rtc-video-quality/generate_graphs.py

```diff
@@ -45,7 +45,7 @@ def split_data(graph_data, attribute):
     if value not in groups:
       groups[value] = []
     groups[value].append(element)
```

Comment:
You could eliminate lines 45 and 46 by either:
- Appending to groups.get(value, []) instead of groups[value]
- Making groups a collections.defaultdict(list)

Reply:
But this isn't part of the conversion to python3. This can be updated in the refactoring process in the next pull request

Reply:
Sure, good point.